### ISSUE TITLE
[TF2] [Feature Request] Show special taunt inspect details in killcam screen

### ISSUE THREAD

**Issue by spritersonikku** [ValveSoftware/Source-1-Games#6425]
Some background first: when a player is killed, it shows details about the currently-held weapon -- assuming that it isn't a stock (gray-named) item. Players can also use the Inspect key to inspect a teammates' cosmetics/weapons in the same way.
Before the Love & War update dropped, taunts were placed in the "Action" slot and thus would appear in the "Player is carrying" toast when you used the Inspect key.
Taunts are now placed into new loadout slots which are invisible to everyone but the owner in-game. Because of this, there is no reason to name or describe taunts, as no other player can see them unless they go to the user's Steam inventory to view it.

So my suggestion is: if a player is using a special taunt during the killcam screen, it will show the Inspect details of that taunt -- the name, description, quality, and any special attributes (i.e. a Strange counter in the case of the Burstchester).
As a secondary suggestion, maybe include the same details about the currently-used taunt to teammates if they use the Inspect key.

**Comment by JoriKos** [comment]
I think the killcam one could be default, but the inspect one could possibly just be a convar instead. If I accidentally miss the weapon I wanna see on someone's inspect, I don't wanna cycle through 16 items.

**Comment by spritersonikku** [comment]
Yeah that's why I figure it may be best to only allow inspecting the actively-used taunt in cases of Inspect key stuff.
